src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -405,9 +410,9 @@ private void ValidateDownloadOptions(MigrateRepoCommandArgs args)
             throw new OctoshiftCliException("Both --ssh-user and --ssh-private-key must be specified for SSH download.");
         }
 
-        if (args.SmbUser.HasValue() ^ args.SmbPassword.HasValue())
+        if ((args.SmbUser.HasValue() && GetSmbPassword(args).IsNullOrWhiteSpace()) || (args.SmbPassword.HasValue() && args.SmbUser.IsNullOrWhiteSpace()))
         {
-            throw new OctoshiftCliException("Both --smb-user and --smb-password must be specified for SMB download.");
+            throw new OctoshiftCliException("Both --smb-user and --smb-password (or SMB_PASSWORD env. variable) must be specified for SMB download.");
```

Comment:
```suggestion
            throw new OctoshiftCliException("Both --smb-user and --smb-password (or `SMB_PASSWORD` environment variable) must be specified for SMB download.");
```

Reply:
will apply the change but two things for consistency:
1. We didn't use back ticks in any of our command docs as far as I know since the help text is a plain text and back-ticks won't do anything special. That being said if you think it'd be better to use them, then we need to apply it to all other help docs too which can be done in a separate PR with a new ticket. 
2. If environment is preferred over env. then we need to change every other occurrence too. Probably we're going to need an issue for it 😃

Reply:
I am going to skip this b/c everywhere else we used `env.` instead of `environment` and no back ticks.